package.json

```diff
@@ -37,8 +37,8 @@
   },
   "homepage": "https://github.com/Bandwidth/shared-components#readme",
   "peerDependencies": {
-    "react": "15.5.x",
-    "react-dom": "15.5.x"
+    "react": ">=15.5.x <= 16.0.0",
```

Comment:
Can we make this `16.0.x` to give ourselves some flexibility to use versions of 16 which have patch changes (usually bugfixes or security issues)?